src/GuppyDataExplorer/ExplorerTable/index.jsx

```diff
@@ -353,6 +375,14 @@ class ExplorerTable extends React.Component {
             </div>
           )}
 
+      <div>
+        <CheckBox
```

Comment:
can you supply a dummy ID to this `CheckBox` since that is a required props for that component